modules/instrument_builder/js/instrument_builder.instrument.js

```diff
@@ -120,11 +120,20 @@ var Instrument = {
 
                         // Add dropdown and special naming when no date format is set
                         // (i.e when addDateElement() is used)
-                        if (element.Options.dateFormat === "") {
+                        if (!element.Options.dateFormat) {
+                          if (elName === '' || !elName.includes('_date')) {
                             elName = elName + "_date";
-                            dropdown = "select{@}" + elName + "_status" +
+                          }
+                          console.log(element.Description);
+                          if (!element.Description) {
+                            element.Description = '';
```

Comment:
same here, i think the description shouldn't be empty?

Reply:
@zaliqarosli I think that was to get rid of an undefined being added to the instrument file.